src/main/java/me/xmrvizzy/skyblocker/skyblock/item/ItemCooldowns.java

```diff
@@ -0,0 +1,114 @@
+package me.xmrvizzy.skyblocker.skyblock.item;
+
+import com.google.common.collect.ImmutableList;
+import me.xmrvizzy.skyblocker.config.SkyblockerConfig;
+import me.xmrvizzy.skyblocker.config.SkyblockerConfigManager;
+import me.xmrvizzy.skyblocker.events.ClientPlayerBlockBreakEvent;
+import me.xmrvizzy.skyblocker.utils.ItemUtils;
+import net.fabricmc.fabric.api.event.player.UseItemCallback;
+import net.minecraft.block.BlockState;
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.Hand;
+import net.minecraft.util.TypedActionResult;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class ItemCooldowns {
+    private static final String JUNGLE_AXE_ID = "JUNGLE_AXE";
+    private static final String TREECAPITATOR_ID = "TREECAPITATOR_AXE";
+    private static final String GRAPPLING_HOOK_ID = "GRAPPLING_HOOK";
+    private static final ImmutableList<String> BAT_ARMOR_IDS = ImmutableList.of("BAT_PERSON_HELMET", "BAT_PERSON_CHESTPLATE", "BAT_PERSON_LEGGINGS", "BAT_PERSON_BOOTS");
+
+    private static final SkyblockerConfig.ItemCooldown config = SkyblockerConfigManager.get().general.itemCooldown;
+    private static final Map<String, CooldownEntry> itemCooldowns = new HashMap<>();
```

Comment:
These two constants don't follow the java naming conventions, also why is config a constant? its only used twice from what I see and I don't think it provides a readability advantage

Reply:
fixed